server/build.gradle

```diff
@@ -81,6 +81,7 @@ dependencies {
     compile group:'com.fasterxml.jackson.core', name:'jackson-core', version:jackson_version
     compile group:'com.fasterxml.jackson.core', name:'jackson-databind', version:jackson_version
     compile group:'com.fasterxml.jackson.core', name:'jackson-annotations', version:jackson_version
+    compile group:'org.jmockit', name:'jmockit', version:jmockit_version
```

Comment:
Why is this change needed?

Won't this result in jmockit being included in the .war?